index.js

```diff
@@ -379,6 +379,10 @@ function signApplicationAsync (opts) {
         args.push('--keychain', opts.keychain)
       }
 
```

Comment:
Probably we can amend this blank line together to keep some consistency I think? The grouping may be slightly clearer if so. 😕